src/runtime/mini_hexagon_dma.h

```diff
@@ -18,17 +18,18 @@ typedef unsigned long addr_t;
 
 typedef unsigned int qurt_size_t;
 typedef unsigned int qurt_mem_pool_t;
+#define ENUM  enum __attribute__((aligned(4)))
 
 __inline static int align(int x,int a) {
     return ( (x+a-1) & (~(a-1)) );    
 } 
 
-enum { QURT_EOK = 0 };
+ENUM { QURT_EOK = 0 };
```

Comment:
The alignment attribute can't possibly matter here (this enum is unnamed/doesn't define a type, the constant itself doesn't have any meaningful alignment).

Reply:
Agree we will change it.